### PR TITLE
Feature/add book

### PR DESCRIPTION
**Pull request by ronlechiht**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -9,5 +9,9 @@ import {
   BorrowingController
 } from './modules/borrowing-returning-book/index.js';
 
-const bookApp = new BookController(new BookModel(), new BookView());
+const bookApp = new BookController(new BookManagerModel(), new BookView());
 const borrowingApp = new BorrowingController(new BorrowingModel(), new BorrowingView());
+
+bookApp.displayBookList();
+bookApp.handleAddNewButton();
+bookApp.handleAddBookForm();
```
Comment (batruongq) [inline src/index.js:17]: Should not handle line 16, 17 here. Event of elements in module should be set in module

File: src/modules/book/book.view.js [hunk 2]
```diff
@@ -39,4 +84,28 @@ export class BookView {
       modal.style.display = 'none';
     }
   }
+
+  bindAddBook(handler) {
+    const addButton = document.querySelector('.add-btn');
+
+    addButton.onclick = () => {
+      const isbn = document.querySelector('input[name="isbn-input"]').value;
```
Comment (cuongngo-agilityio) [inline src/modules/book/book.view.js:92]: Move query element to constructor. Query once and using. If you do the query here, you get the performance issue (query multi times every click)

File: src/modules/book/book.view.js [hunk 3]
```diff
@@ -39,4 +84,28 @@ export class BookView {
...
+        name: name,
+        author: author,
+        issued: issued
+      }
+
+      if (this.validateForm(book)) {
+        handler(book);
+        this.renderAddedBook(book);
+        this.resetInput();
+      }
```
Comment (cuongngo-agilityio) [inline src/modules/book/book.view.js:108]: Do the validation on controller. Controller handle business, view just work with UI

File: src/modules/book/book.view.js [hunk 4]
```diff
@@ -39,4 +84,28 @@ export class BookView {
...
   }
+
+  bindAddBook(handler) {
+    const addButton = document.querySelector('.add-btn');
+
+    addButton.onclick = () => {
+      const isbn = document.querySelector('input[name="isbn-input"]').value;
+      const name = document.querySelector('input[name="name-input"]').value;
+      const author = document.querySelector('input[name="author-input"]').value;
+      const issued = document.querySelector('input[name="issued-input"]').value;
```
Comment (cuongngo-agilityio) [inline src/modules/book/book.view.js:95]: Move query element to constructor. Query once and using. If you do the query here, you get the performance issue (query multi times every click)

**Pull request merged**